tests/util/test_ssl.py

```diff
@@ -56,3 +57,28 @@ def test_ssl_context_caching() -> None:
     assert create_no_verify_ssl_context() is create_no_verify_ssl_context(
         SSLCipherList.PYTHON_DEFAULT
     )
+
+
+def test_cteate_client_context(mock_sslcontext) -> None:
```

Comment:
cteate -> create

Reply:
Ah my bad, sorry for the typo. How should I correct this now the PR has been merged?

Reply:
Please create a new PR with the fix for the typo.

Reply:
PR created as #142799